schemas/content/product.schema.json

```diff
@@ -60,81 +53,68 @@
-          "type": "string",
-          "description": "The description of the product."
-        },
-        "xdm:fabrication": {
-          "title": "Fabrication",
-          "type": "string",
-          "description": "Primary material the product is constructed with."
-        },
-        "xdm:gender": {
-          "title": "Gender",
-          "type": "string",
-          "description": "Gender the product is marketed to.",
-          "meta:enum": {
-            "male": "For men",
-            "female": "For women",
-            "unisex": "For both men and women",
-            "unknown": "Unknown"
-          }
-        },
-        "xdm:size": {
-          "title": "Size",
-          "type": "number",
-          "description": "Standard product size for the product."
-        },
-        "xdm:unitOfMeasure": {
-          "title": "Unit of Measure",
-          "type": "string",
-          "description":
-            "Standard unit of measure of the variant. Denotes the units for the size measurement."
-        },
-        "xdm:countryOfOrigin": {
-          "title": "Country of Origin",
-          "type": "string",
-          "pattern": "^[A-Z]{2}$",
+            "The prices for this product. In the `currencyCode` currency."
+        },
+        "xdm:assets": {
+          "title": "Assets",
+          "type": "array",
+          "items": {
+            "$ref":
+              "https://ns.adobe.com/xdm/assets/asset#/definitions/asset"
+          },
           "description":
-            "The two-character [ISO 3166-1 alpha-2](https://datahub.io/core/country-list) code for the country of origin of the product as defined by customs requirements."
-        },
-        "xdm:COGS": {
-          "title": "Cost of Good Sold",
-          "type": "number",
-          "description": "Cost of Good Sold. In the `currencyCode` currency."
-        },
-        "xdm:listPrice": {
-          "title": "List Price",
-          "type": "number",
-          "description":
-            "Default price of the product before sales and discounting. In the `currencyCode` currency."
-        },
-        "xdm:currencyCode": {
-          "title": "Currency Code",
-          "type": "string",
-          "examples": ["USD", "EUR"],
-          "pattern": "^[A-Z]{3}$",
+            "List of assets associated with this product/variant."
+        },
+        "xdm:attributes": {
+          "title": "Attributes",
+          "type": "array",
+          "items": {
+            "$ref":
+              "https://ns.adobe.com/xdm/content/product-attribute#/definitions/attribute"
+          },
           "description":
-            "The ISO 4217 alphabetic currency code used for cost and pricing, including `listPrice` and `COGS`."
-        },
-        "xdm:originalSaleDate": {
-          "title": "Original Sale Date",
-          "type": "string",
-          "format": "date",
+            "List of attributes associated with this product/variant."
```

Comment:
We seem a bit inconsistent in the descriptions where we refer to product, product/variant etc. Can you do a pass please and ensure we are consistent please.